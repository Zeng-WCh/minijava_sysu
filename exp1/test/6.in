1-4-9+4-4-5-7-0-4-7+6+6-0+5+4+4+4+2+9-4-2+2-3-1+2+8+9+0-6-7-5+1-2-7+8-8-9+7-4+9-0+9-9-3-5-4+9+2-4-1-0-9-0+2+2+6-8+7-3-9-6-0-6+8+9-0+7-8+9+3+8-7-7+1-7-4-6-4+7+5-2+0+8-8-3+6+3+9-7+2-9+0-4+0-8+7+0+6-5-6