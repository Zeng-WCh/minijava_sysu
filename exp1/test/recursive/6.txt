2-3-5-1-4+9-5-6-8-8-9+6-9-8-8-1-2+1+8-5-1-7-8+6+6-8-4+3-5+7