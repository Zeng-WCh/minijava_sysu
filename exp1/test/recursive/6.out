Input an infix expression and output its postfix notation:
30-9-1+9+6+1-2-8+3-9+6+0+9-2-0+8+2-4-3+5-1+9+6+9+7-1-1-4-9-
End of program.
